src/lib/es2015.proxy.d.ts

```diff
@@ -1,5 +1,5 @@
 interface ProxyHandler<T> {
-    getPrototypeOf? (target: T): any;
+    getPrototypeOf? (target: T): Object | null;

```

Comment:
Done. Still I wonder whether is using `{}` clearer than `Object`? I know that these two are confusing for newcomers. But at least `Object` suggests non-primitive values.